index.js

```diff
@@ -4,10 +4,26 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "red"
 
 /***** Deliverable 3 *****/
-// console.log("PLAYERS array looks like this:", PLAYERS)
+console.log("PLAYERS array looks like this:", PLAYERS)
+
+
+PLAYERS.forEach(function (player) {  
+    const newDiv = document.createElement("div");
+    newDiv.classList.add("player")
+    newDiv.setAttribute("data-number", player.number)
+    document.body.appendChild(newDiv);
+    const h3 = document.createElement("h3")
+    h3.innerHTML = `${player.name} (<em>${player.nickname}</em>)`
+    newDiv.appendChild(h3)
+    newDiv.innerHTML = `<img src="${player.photo}" alt="${player.name}">`
+
+});
```

Comment:
Nice! I know we went over this in lecture, but just to put it in the comment too - on line 21, calling `newDiv.innerHTML = ` will over-write whatever's currently inside the div, so the `h3` that you created above will go away. Instead, you can use `newDiv.innerHTML = ` to create the h3 and the image:

```js
newDiv.innerHTML = `
<h3>${player.name} (<em>${player.nickname}</em>)</h3>
<img src="${player.photo}" alt="${player.name}">
`
```